css-scroll-snap-2/Overview.bs

```diff
@@ -275,119 +275,129 @@ Snap Events {#snap-events}
 	<table class="data" id="eventhandlers">
 		<thead>
 			<tr>
-				<th>Event handler
-				<th>Event handler event type
+				<th>Event</th>
+				<th>Interface</th>
+				<th>Targets</th>
+				<th>Description</th>
+			</tr>
+		</thead>
 		<tbody>
 			<tr>
-				<th><dfn event>snapChanged</dfn>
-				<td>{{scroll!!event}}
+				<th><dfn for="snapchanged" event>snapchanged</dfn></th>
+				<td>{{SnapEvent}}</td>
+				<td>scroll containers</td>
+				<td>Fired at the scrolling element or {{Document}} at the end of a scroll (before a {{scrollend}} event)
+					or after a <a href="https://drafts.csswg.org/css-scroll-snap-1/#re-snap">layout snap</a>
+					if the element that the scrolling element or Document is snapped to changed.</td>
+			</tr>
 			<tr>
-				<th><dfn event>snapChanging</dfn>
-				<td>{{scroll!!event}}
+				<th><dfn for="snapchanging" event>snapchanging</dfn></th>
+				<td>{{SnapEvent}}</td>
+				<td>scroll containers</td>
+				<td>Fired at the scrolling element or {{Document}} during scrolling (before a {{scroll}} event),
+					if the scrolling would lead to a change in the
+					element that the scrolling element or Document is snapped to.</td>
```

Comment:
I think we want to explicitly call out that it is also fired if it's different than the last time the event was fired. E.g. if you scroll over a new potential target and scroll back, it should fire another event to tell you you are back on the original element even though its the same as the element the document is snapped to.

Reply:
Yes, in fact I think the case you describe is actually all we need - what I had before ("a change in the	element that the scrolling element or Document is snapped to.")  implies that snapchanging would keep firing as long as you are scrolling over a target that isn't the last snapped-to target, which isn't the intent. I've adjusted the wording.